src/main/java/Duke.java

```diff
@@ -1,10 +1,189 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws DukeException {
```

Comment:
The main method is quite long imo. Would it be possible to extract some code into other classes/methods?